libs/ui/date-picker/helm/src/lib/hlm-date-picker.component.ts

```diff
@@ -0,0 +1,138 @@
+import { BooleanInput } from '@angular/cdk/coercion';
+import { booleanAttribute, Component, computed, forwardRef, input, model, output, signal } from '@angular/core';
+import { NG_VALUE_ACCESSOR } from '@angular/forms';
+import { NgIcon, provideIcons } from '@ng-icons/core';
+import { lucideCalendar } from '@ng-icons/lucide';
+import { hlm } from '@spartan-ng/brain/core';
+import { type ChangeFn, type TouchFn } from '@spartan-ng/brain/forms';
+import { BrnPopoverComponent, BrnPopoverContentDirective, BrnPopoverTriggerDirective } from '@spartan-ng/brain/popover';
+import { HlmCalendarComponent } from '@spartan-ng/ui-calendar-helm';
+import { HlmIconDirective } from '@spartan-ng/ui-icon-helm';
+import { HlmPopoverContentDirective } from '@spartan-ng/ui-popover-helm';
+import type { ClassValue } from 'clsx';
+import { injectHlmDatePickerConfig } from './hlm-date-picker.token';
+
+export const HLM_DATE_PICKER_VALUE_ACCESSOR = {
+	provide: NG_VALUE_ACCESSOR,
+	useExisting: forwardRef(() => HlmDatePickerComponent),
+	multi: true,
+};
+
+@Component({
+	selector: 'hlm-date-picker',
+	standalone: true,
+	imports: [
+		NgIcon,
+		HlmIconDirective,
+		BrnPopoverComponent,
+		BrnPopoverTriggerDirective,
+		BrnPopoverContentDirective,
+		HlmPopoverContentDirective,
+		HlmCalendarComponent,
+	],
+	providers: [HLM_DATE_PICKER_VALUE_ACCESSOR, provideIcons({ lucideCalendar })],
+	template: `
+		<brn-popover sideOffset="5" closeDelay="100">
+			<button type="button" [class]="_computedClass()" [disabled]="state().disabled()" brnPopoverTrigger>
+				<ng-icon hlm size="sm" name="lucideCalendar" />
+
+				@if (formattedDate(); as formattedDate) {
+					{{ formattedDate }}
+				} @else {
+					<ng-content />
+				}
+			</button>
+
+			<div hlmPopoverContent class="w-auto p-0" *brnPopoverContent="let ctx">
+				<hlm-calendar
+					[date]="date()"
+					[min]="min()"
+					[max]="max()"
+					[disabled]="state().disabled()"
+					(dateChange)="_handleChange($event)"
+				/>
+			</div>
+		</brn-popover>
+	`,
+	host: {
+		class: 'block',
+	},
+})
+export class HlmDatePickerComponent<T> {
+	private readonly _config = injectHlmDatePickerConfig<T>();
+
+	public readonly userClass = input<ClassValue>('', { alias: 'class' });
+	protected readonly _computedClass = computed(() =>
+		hlm(
```

Comment:
There appears to be a border and a box shadow in our implementation which differs from the shadcn implementation:

<img width="1208" alt="image" src="https://github.com/user-attachments/assets/98c692de-863a-445d-825a-e41df619259f" />

Reply:
Good catch! I checked the shadcn/ui styles and it looks like the calendar only receives border styles when used in line. I override the border styles for the date-picker calendar, thus only the styles from the popover content are applied.